TUTORIAL.md

```diff
@@ -572,29 +584,40 @@ Flow(
 
 
 
-Previous flow was a bit complicated, but luckily we have the `join`, `concatenate` and `filter_rows` processors which make such combinations a snap
+Previous flow was a bit complicated, but luckily we have the `join`, `concatenate` and `filter_rows` processors which make such combinations a snap:
 
 
 ```python
-from dataflows import Flow, load, dump_to_path, join, concatenate, filter_rows, printer, checkpoint
+from dataflows import (
+    Flow,
+    load,
+    dump_to_path,
+    join,
+    concatenate,
+    filter_rows,
+    printer,
+    checkpoint,
+)
 
 Flow(
     # load from the checkpoint we saved in the previous flow
-    checkpoint('emmy-academy-nominees-winners'),
-    filter_rows(equals=[dict(winner=1)], resources=['emmies']),
+    checkpoint("emmy-academy-nominees-winners"),
+    filter_rows(equals=[dict(winner=1)], resources=["emmies"]),
     concatenate(
-        dict(emmy_nominee=['nominee'],),
-        dict(name='emmies_filtered'),
-        resources='emmies'
+        dict(emmy_nominee=["nominee"],),
+        dict(name="emmies_filtered"),
+        resources="emmies",
```

Comment:
Same reasoning here with the extra comma. [Black](https://github.com/psf/black), the Python code formatter, automatically adds those.